nameserver/nameserver.go

```diff
@@ -208,8 +208,8 @@ func (n *Nameserver) receiveGossip(msg []byte) (mesh.GossipData, mesh.GossipData
 		return n.isKnownPeer(e.Origin)
 	})
 
+	var entriesToBroadcast []Entry
```

Comment:
it would be nice if that variable name described what those entries *are* rather than what we end up doing with them.

`newOrUpdatedEntriesOriginatingFromUs`. urgh.